cmd/ocm-backplane/console/console.go

```diff
@@ -693,3 +694,26 @@ func loadConsolePlugins(config *rest.Config) (string, error) {
 
 	return consolePlugins, nil
 }
+
+// Get pull secret file saving path, default to ~/.kube/ocm-pull-secret folder
+func GetConfigDirectory() (string, error) {
+	if configDirectory == "" {
+		home, err := homedir.Dir()
+		if err != nil {
+			return "", fmt.Errorf("can't get user homedir. Error: %s", err.Error())
+		}
+
+		// Define directory and filename for config
+		configDirectory := filepath.Join(home, ".kube/ocm-pull-secret")
+
+		return configDirectory, nil
+	}
+
+	return configDirectory, nil
+}
+
+// Modify pull secret file saving path
+func ModifyConfigDirectory(basePath string) error {
```

Comment:
I am not quite understand this function. 
It is used in test only. And cannot we set it as given value to a variable directly? Why a function is needed.

Reply:
In that case, the variable need to be public ( aka `ConfigDirectory` ) since it calls by other modules, I guess having a public function is a good approach rather than modifying the variable.

Reply:
Can this be moved to `utils` then? As it is a utility function?

Reply:
This can be made into a global var once the naming comment above has been addressed. I see no issues in it rather than having a function do something redundant.

Reply:
Remove the function and only depend on the `pullSecreConfigDirectory` variable